hip-config-hcc.cmake.in

```diff
@@ -42,8 +54,8 @@ endif()
 set(HIP_COMPILER "@HIP_COMPILER@")
 set(HIP_RUNTIME "@HIP_RUNTIME@")
 
-set_and_check( hip_INCLUDE_DIR "@PACKAGE_INCLUDE_INSTALL_DIR@" )
-set_and_check( hip_INCLUDE_DIRS "${hip_INCLUDE_DIR}" )
+set_and_check( hip_INCLUDE_DIR "@PACKAGE_INCLUDE_INSTALL_DIR@")
+set( hip_INCLUDE_DIRS "${hip_INCLUDE_DIR}" "${HSA_PATH}/include" )
```

Comment:
Leave `set_and_check` at least on `hip_INCLUDE_DIRS`, as long as you suggest that `HSA_PATH` is verified.

Reply:
Added check for HSA include path